### ISSUE TITLE
After upgrade to 0.88.2 realm is not generating the CacheData class for the Proxy classes

### ISSUE THREAD

**Issue by emanuelet** [realm/realm-java#2519]
So after I made the upgrade to the new version I get the following error for all the Proxy classes when I try to build the app on a device: "error: cannot find symbol class CacheData".

I tried with both the 0.88.2 and the 0.89.0 version but the problem doesn't change.
I also try to delete the build folder to force a rebuild and it din't help either.

Any ideas why this could happen?

Realm version(s): 0.88.2/0.89.0-SNAPSHOT
OS: Ubuntu Linux 15.10
Gradle: 2.11
Android Studio version: 1.5.1

**Comment by stk1m1** [comment]
@emanuelet Hi, could you share your Gradle file with us? It is also helpful to have a stacktrace as well. JFYI, the latest release is [v0.88.2](https://github.com/realm/realm-java/releases/tag/v0.88.2).

**Comment by emanuelet** [comment]
Hi,
so in the end I found what the problem was.
I was adding correctly the 0.88.2 version but I had 2 Gradle dependecies that were still requesting the older version (com.uphyca:stetho_realm:0.8.0 and com.github.thorbenprimke:realm-recyclerview:0.9.5).
Temporarirly I have reverted back to the old version, as I can wait for the libraries to be updated, otherwise I will consider forking them.

@stk1m1 Thanks for your time.
